packages/app/ui/beat-list-entry/index.tsx

```diff
@@ -16,14 +16,16 @@ export function BeatListEntry({
   onPress: () => void;
 }) {
   return (
-    <Pressable onPress={onPress}>
+    <TextLink href={`/dashboard/search/beat/${entry.id}`}>
```

Comment:
doing this will prevent users from playing the beat, so the link should go under show more, similar how Spotify does it.